lib/rubygems/commands/setup_command.rb

```diff
@@ -374,8 +377,15 @@ def install_default_bundler_gem
         each {|default_gem| rm_r File.join(bundler_spec.gems_dir, default_gem) }
     end
 
-    mkdir_p bundler_spec.bin_dir
-    bundler_spec.executables.each {|e| cp File.join("bundler", bundler_spec.bindir, e), File.join(bundler_spec.bin_dir, e) }
+    bundler_bin_dir = File.join(
+      options[:destdir], Gem.default_dir, 'gems',
```

Comment:
I searched in RubyGems source code, but could not find a method or
constant that returns this gems directory, it seems hardcoded at other
places too.
In Ruby source code a literal string is used too if I'm correct:
https://svn.ruby-lang.org/cgi-bin/viewvc.cgi/tags/v2_4_2/tool/rbinstall.rb?revision=59899&view=markup#l724